src/components.jl

```diff
@@ -303,6 +303,17 @@ function scalefontsizes(factor::Number)
     end
 end
 
+"Resets font sizes to initial default values"
```

Comment:
This doctext will not display properly - when you `?scalefontsizes` at the REPL it will print out all the docstrings, but not which methods they're associated with. So you need to do something like
"""
    scalefontsizes([x])
Increase the default font size by a factor `x`. The non-argument form restores the defaults
"""